### PR TITLE
distro selection docs

### PR DESCRIPTION
**Pull request by jeremydvoss**
Updating website docs for auto instrumentation with respect this this pr: https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1823

### CODE REVIEW

**Review by svrnm — CHANGES_REQUESTED** [review]

**Comment by cartermp** [comment]
The other thing is that I don't like having to list out all of these options unless necessary. Are there defaults (like `distro`) that will be there when unspecified?

File: content/en/docs/instrumentation/python/automatic/agent-config.md
Comment (aabmass) [reply content/en/docs/instrumentation/python/automatic/agent-config.md:36]: +1 even as someone involved I don't remember what distro vs configurator is. Would be nice to have something explaining what they are.

Comment (ocelotl) [reply content/en/docs/instrumentation/python/automatic/agent-config.md:36]: We do have documentation (at least for distros) [here](https://opentelemetry.io/docs/instrumentation/python/distro/). I'm not sure this is the place where we should document what a distro or a configurator is. We don't do any explanation for `--traces_exporter` or `--metrics_exporter` either.

Maybe just a link to the general documentation would be enough here?

Comment (svrnm) [reply content/en/docs/instrumentation/python/automatic/agent-config.md:36]: I always found the distros page you linked confusing because it replicates a lot of what is written in the automatic documentation, so we need to merge them eventually. Also, Python is the only language as far as I am aware that has this understanding of a distribution. 

I have no issue with `distro` and `configurator` being documented as outlined here, but what we need is examples what options I have as an enduser, if there are currently only 1 option and they will come later, then let's postpone this for now and add it when people have a choise.

> We don't do any explanation for --traces_exporter or --metrics_exporter either.

We could have them.

**Comment by svrnm** [comment]
@open-telemetry/python-maintainers / @open-telemetry/python-approvers I leave it to you if/when we add this to the docs, please provide guidance

**Comment by jeremydvoss** [comment]
> The other thing is that I don't like having to list out all of these options unless necessary. Are there defaults (like `distro`) that will be there when unspecified?

Yes the base distro that can also be referred to as the "distro" entry point is the default.

**Comment by cartermp** [comment]
@jeremydvoss sorry, I'm not sure I understand. Is the option `--distro` a required parameter when running the autoinstrumentation agent?

**Comment by cartermp** [comment]
Closing due to inactivity

**Pull request closed without merging**
